nginx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nginx:alpine
+
+RUN \
+    apk add --update curl && \
+    addgroup -S www-data && \
+    adduser -H -h /static -S -G www-data -s /bin/false www-data
+RUN rm -rf /etc/nginx/conf.d/*
+ADD nginx.conf /etc/nginx/nginx.conf
```

Comment:
Same here add a single line for the RUN in line 3 and 7

Reply:
Normally the "rm -rf" should be part of the same layer that adds those files. But this is a special case, as these files are already within the base image "nginx:alpine", so we cannot delete them from the image at all, but only "mark them as deleted" in a new layer.
combining it with the former layer does therefore not make sense. One could argue, that it should be in front of the software update/installation, because it removes things from the base image and the software update step is expected to change more frequently than the rm or the base image. Therefore the update should be further down, to better utilize layer caching for building images. But it does not really matter to the user that much...